lib/mrsk/configuration.rb

```diff
@@ -153,7 +153,20 @@ def ssh_proxy
   end
 
   def ssh_options
-    { user: ssh_user, proxy: ssh_proxy, auth_methods: [ "publickey" ] }.compact
+    { user: ssh_user, proxy: ssh_proxy, auth_methods: [ "publickey" ], keepalive: true, keepalive_interval: 30 }.compact
+  end
+
+
+  def sshkit_max_concurrent_starts
+    raw_config.sshkit["max_concurrent_starts"] if raw_config.sshkit.present?
+  end
+
+  def sshkit_pool_idle_timeout
+    if raw_config.sshkit.present?
+      raw_config.sshkit["pool_idle_timeout"] || 900
+    else
+      900
+    end
```

Comment:
Since there are already a bunch of different SSH configuration methods, I think it would be better to extract them to a separate class similar to `Mrsk::Configuration::Accessory`, `Mrsk::Configuration::Builder` and others.

Reply:
I've extracted ssh and sshkit configuration classes here - https://github.com/mrsked/mrsk/pull/399/commits/94d6a763a8895c3d5250470e92d4223d7fecab0e

Reply:
Looks much cleaner now.